src/components/historyData.js

```diff
@@ -0,0 +1,82 @@
+import React, { PureComponent, PropTypes } from 'react'
+import { connect } from 'react-redux'
+import reactCSS from 'reactcss'
+
+const styles = reactCSS({
+  default: {
+    historyData: {
+      float: 'right',
+      width: '49.8%',
+      marginBottom: 0,
+      backgroundColor: '#3ed3c7',
+      height: 240,
+    },
+    container: {
+      float: 'top',
```

Comment:
`top` is not a valid value for `float` https://developer.mozilla.org/en-US/docs/Web/CSS/float#Values